format/index.md

```diff
@@ -300,6 +300,11 @@ pagination links, as described below.
 If a link object refers to resource objects included in the same compound
 document, it **MUST** include resource linkage to those resource objects.
 
+> Note: a resource may actually be referring to another resource returned 
+in the same compound document, but this reference may be omitted 
+from the link object of the parent resource because of sparse fieldset 
+restrictions.
+
```

Comment:
> the link object *simply wouldn't appear* if the field has been excluded

My thoughts exactly. An excluded field is supposed to be dropped from `links` altogether, so the note is redundant.